### ISSUE TITLE
Access Keys not working?

### ISSUE THREAD

**Issue by hoergen** [friendica/friendica#3840]
I stumbled over the access Keys article in the help https://horche.demkontinuum.de/help/Accesskeys because I searched a quick way to switch between accounts, instead of clicking-pulldown-clicking management-waiting-clicking-waiting-there you go.
Didnt find a solution for my idea, but couldn't get those access keys to work (linux&firefox Alt+Shift+<acc key>)

Are they still working?

**Comment by MrPetovan** [comment]
Yes, I can scroll my feed using J and K directly. Not sure about any other though.

**Comment by hoergen** [comment]
You are talking about navigation keys. but the article says something about keys where you are able to jump to different pages and tabs.

**Comment by MrPetovan** [comment]
Indeed, my mistake. There aren't any access keys I can see in frio, I'm afraid.

**Comment by MrPetovan** [comment]
I just successfully tested the access keys of the vier theme in Firefox, access keys aren't implemented in the Vivaldi browser for instance.

**Comment by tobiasd** [comment]
So this is a problem/feature request for the frio theme?

**Comment by MrPetovan** [comment]
It sounds like it indeed.
